easybuild/tools/options.py

```diff
@@ -480,6 +480,8 @@ def override_options(self):
                          'int', 'store', None),
             'parallel-extensions-install': ("Install list of extensions in parallel (if supported)",
                                             None, 'store_true', False),
+            'prefer-ebpythonprefix-over-pythonpath': ("Replaces PYTHONPATH with EBPYTHONPREFIX when possible",
```

Comment:
`--prefer-ebpythonprefixes` is specific enough, I think

Reply:
We bounced around a bunch of different names, so i'm currently thinking maybe `--replace-pythonpath` is good enough. The EBPYTHONPREFIXES name is kind of an internal implementation detail in easybuild which wouldn't mean much to any user. The comment and documentation could go into more depth to explain it and how it connects to the site customize script our python installs have.

The fact that it doesn't always replace pythonpath, exceptions really are very rare, and this wouldn't be the first option that does what is claims only when it is supported.